### DISCUSSION TITLE
How to use Flutter code with python and flet

### DISCUSSION THREAD

**Discussion by imadityacoder** [flet-dev/flet#4600] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

I want to use python libaries with flutter code directly together. How can I do it.

**Comment by ndonkoHenri** [comment]
You might find [serious-python](https://github.com/flet-dev/serious-python) interesting.

**Reply by joesandiroz2** [reply]
> You might find [serious-python](https://github.com/flet-dev/serious-python) interesting.

can i use flutter third party in pub.dev . for example there is this package [https://pub.dev/packages/calendar_date_picker2 ](url) can I use that package directly in flet. can you give an example

**Reply by ndonkoHenri** [reply]
We have a guide on that here: https://flet.dev/docs/extend/integrating-existing-flutter-packages

**Comment by imadityacoder** [comment]
Can I use serious-python like flutter and python codes parellely?
